specs/experimental/op-contracts-manager.md

```diff
@@ -20,146 +20,247 @@ of governance approved [contract releases] can be found on the
+The OP Contracts Manager refers to a series of contracts, of which a new singleton is deployed
+for each new release of the OP Stack contracts.
 
-TODO.
+The OP Contracts Manager corresponding to each release can be used to:
 
-## Interface
+1. Deploy a new OP chain.
+2. Upgrade the contracts for an existing OP chain from the previous release to the new release.
+3. Orchestrate adding a new game type on a per-chain basis
 
-Version 1.0.0 of the OP Contracts Manager deploys the `op-contracts/v1.6.0`
-contracts release.
+## Getter Methods
 
-### `Proxy.sol`
+The following interface defines the available getter methods:
 
-The OP Contracts Manager is a proxied contract using the standard `Proxy.sol` contract that lives in
-the Optimism monorepo. Therefore the OP Contracts Manager will have the same interface as the
-`Proxy.sol`, in addition to other methods defined in this specification.
+```solidity
+/// @notice Returns the latest approved release of the OP Stack contracts.
+/// @notice Release strings follow semver and are named with the
+/// format `op-contracts/vX.Y.Z`.
+function latestRelease() external view returns (string memory);
 
-The privileged methods of the OP Contracts Manager will be held by the L1 ProxyAdmin owner, as
-specified by the [standard configuration].
+    /// @notice Represents the interface version so consumers know how to decode the DeployOutput struct
+    function OUTPUT_VERSION() external view returns (uint256);
+    /// @notice Addresses of the Blueprint contracts.
+    function blueprints() external view returns (Blueprints memory);
+    /// @notice Maps an L2 chain ID to an L1 batch inbox address
+    function chainIdToBatchInboxAddress(uint256 _l2ChainId) external pure returns (address);
+    /// @notice Addresses of the latest implementation contracts.
+    function implementations() external view returns (Implementations memory);
+    /// @notice L1 smart contracts release deployed by this version of OPCM.
+    function l1ContractsRelease() external view returns (string memory);
+    /// @notice Address of the ProtocolVersions contract shared by all chains.
+    function protocolVersions() external view returns (address);
+    /// @notice Address of the SuperchainConfig contract shared by all chains.
+    function superchainConfig() external view returns (address);
+    /// @notice Maps an L2 Chain ID to the SystemConfig for that chain.
+    function systemConfigs(uint256) external view returns (address);
+    /// @notice Semver version specific to the OPContractsManager
+    function version() external view returns (string memory);
+
+/// @notice Returns the implementation contract addresses.
+function implementations() public view returns (Implementations memory);
 
-### `deploy`
+/// @notice Maps an L2 Chain ID to the SystemConfig address for that chain.
+/// @notice All contracts for a chain can be found from its SystemConfig.
+function systemConfigs(uint256 chainId) external view returns (SystemConfig);
```

Comment:
Some arguments have prefixed underscore, others do not

Reply:
Good catch. There were also duplicate methods. 
178e253dd42414bd3a5cb2c8782435953b54f110